DMCompiler/Compiler/DMPreprocessor/DMPreprocessorParser.cs

```diff
@@ -314,6 +315,38 @@ internal static class DMPreprocessorParser {
 
                     return _defines!.ContainsKey(definedInner.Text) ? 1.0f : 0.0f;
                 }
+                else if (token.Text == "fexists") {
+                    Advance();
+                    if (!Check(TokenType.DM_LeftParenthesis)) {
+                        Error("Expected '(' to begin fexists() expression");
+                        return DegenerateValue;
+                    }
+
+                    Token fexistsInner = Current();
+
+                    if (fexistsInner.Type != TokenType.DM_Resource && fexistsInner.Type != TokenType.DM_ConstantString) {
+                        Error($"Unexpected token {fexistsInner.PrintableText} - resource path expected");
```

Comment:
`#if fexists()` doesn't take a resource path as an argument.
```suggestion
                    if (fexistsInner.Type != TokenType.DM_ConstantString) {
                        Error($"Unexpected token {fexistsInner.PrintableText} - file path expected");
```